apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/ListDatasetsOptions.cs

```diff
@@ -41,6 +41,13 @@ public sealed class ListDatasetsOptions
         /// </summary>
         public bool? IncludeHidden { get; set; }
 
+        /// <summary>
+        /// If set, this is the token is used to indicate a continued list operation.
```

Comment:
nit:
this is the token that is used 
otr
this token is used

Reply:
Thanks. Will change to "this token is used" throughout.